python/looker_sdk/rtl/transport.py

```diff
@@ -76,15 +75,14 @@ class TransportOptions(TypedDict, total=False):
     """
 
     timeout: int
-    headers: Optional[MutableMapping[str, str]]
+    headers: MutableMapping[str, str]
```

Comment:
type fix: `Optional` is redundant given the `total=False` above (equivalent to typescript's `Partial`)

Reply:
Good to know. Appreciate the play by play.